cloudinary-core/src/main/java/com/cloudinary/provisioning/Account.java

```diff
@@ -336,15 +335,16 @@ public ApiResponse createUser(String name, String email, Role role, List<String>
      * @param name           Required. Username.
      * @param email          Required. User's email.
      * @param role           Required. User's role.
+     * @param enabled        Optional. User's status (enabled or disabled).
      * @param subAccountsIds Optional. Sub-accounts for which the user should have access.
      *                       If not provided or empty, user should have access to all accounts.
      * @param options        Generic advanced options map, see online documentation.
      * @return The newly created user details.
      * @throws Exception If the request fails.
      */
-    public ApiResponse createUser(String name, String email, Role role, List<String> subAccountsIds, Map<String, Object> options) throws Exception {
+    public ApiResponse createUser(String name, String email, Role role, Boolean enabled, List<String> subAccountsIds, Map<String, Object> options) throws Exception {
```

Comment:
It looks like you just changed the original function (opposed to `updateUser` , where you introduced an overload).

It would lead to compilation issues for some users.